lib/runner/TestRunner.js

```diff
@@ -264,4 +266,16 @@ module.exports = class TestRunner {
         });
       }
     }
+
+    getInvoker(testSuite) {
+        const type = testSuite.type;
+        const platform = testSuite.platform;
+        if (type === CONSTANTS.TYPE.e2e) return CONSTANTS.INVOKER.virtualDeviceInvoker;
```

Comment:
it's really difficult to follow this validations, like this, use explicit brackets for this method.